src/Services/BackgroundService.ts

```diff
@@ -1,51 +1,54 @@
 // BackgroundService.ts
 // Background script that handles the context menu visibility
 
+import browser, { Tabs, Menus } from 'webextension-polyfill'
```

Comment:
Hi,

Sorry, the imports was done by the ide and I failed to check them.

I updated the code to allow development tests again. I added a new file `Browser.ts` that will export `chrome` or `webextension-polyfill` browser based on the environment without loosing the type check. But as a result, some variables  have been changed  to use union types like :
```js
info : Menus.OnClickData | chrome.contextMenus.OnClickData
```
or
```js
let tabInfo : Tabs.Tab[] | chrome.tabs.Tab[]
```

The development debugging with `yarn start` will only work with Chrome though as neither `chome` nor `browser` is available on Firefox.

I hope this will meet your expectations.